website/docs/r/vpc_ipam_pool.html.markdown

```diff
@@ -78,7 +78,7 @@ This resource supports the following arguments:
 within the CIDR range in the pool.
 * `aws_service` - (Optional) Limits which AWS service the pool can be used in. Only useable on public scopes. Valid Values: `ec2`.
 * `description` - (Optional) A description for the IPAM pool.
-* `ipam_scope_id` - (Optional) The ID of the scope in which you would like to create the IPAM pool.
+* `ipam_scope_id` - (Required) The ID of the scope in which you would like to create the IPAM pool.
```

Comment:
This likely needs to be updated too.
https://github.com/hashicorp/terraform-provider-aws/blob/main/internal/service/ec2/ipam_pool.go#L90